marda_registry/data/extractors/galvani.yml

```diff
@@ -0,0 +1,34 @@
+---
+id: >-
+    galvani
+name: >-
+    Galvani
+description: >-
+    Read proprietary file formats from electrochemical test stations
+supported_filetypes:
+    - id: biologic-mpr
+      description: >-
+          Note: support for MPR files created by ECLab >= 11.50 requires 
+          `galvani>0.3`.
```

Comment:
Is that the only supported `filetype` (except arbin that you mentioned in the main comment)?

Reply:
There's also mpt -- haven't tested it before though (will do before merging this)

Reply:
NB this runs up against https://github.com/marda-alliance/metadata_extractors_schema/issues/49, as the usage string will need to be different

Reply:
I'm going to leave mpt off as it returns a Python view into the file, rather than a parsed object itself.